start-here.mdx

```diff
@@ -1,8 +1,153 @@
+
+![Statuses overview](/resources/images/states.png)
+
+It's easiest to visualize the blockchain as a set of [accounts](/foundations/addresses/overview). Each account has an address and a status.
+
+Over its lifetime, an account changes its [status](/foundations/status) among four values:
+
+- `nonexist`: There wasn't a single operation with the account, or it was removed. It has neither a balance, nor code.
+- `uninit`: If some Toncoin is transferred to an account, it now exists, but there is still no smart contract code on it. It now has a balance.
+- `active`: After a deploy message (see below) with code and initial data is sent to an account, it becomes active and can process other messages. It now has a balance, code, and internal state.
+- `frozen`: If an account is overdue on its storage fees, it will be frozen until the fees are paid. If the balance becomes too small, the account goes completely bankrupt, is removed, and ceases to exist.
+
+The _code_ on an active account is a smart contract. The term _"contract"_ is often used for an account that holds the code.
+
+[Internal address](/foundations/addresses/overview#internal-addresses) of an account is a 256-bit number and is usually [formatted](/foundations/addresses/formats) like this: `EQDKbjIcfM6ezt8KjKJJLshZJJSqX7XOA4ff-W72r5gqPrHF`.
+
+There are also [external addresses](/foundations/addresses/overview#external-addresses) for things outside the blockchain. In any situation where an external address is involved, a special "none" address may be provided.
+
+![Blockchain overview](/resources/images/blockchain.png)
+
+Addresses specify where [messages](/foundations/messages/overview) should be delivered. There are three types of messages:
+
+- [internal](/foundations/messages/internal) messages are sent between accounts;
+- [incoming external](/foundations/messages/external-in) messages are sent from code outside the blockchain to a contract;
+- [outgoing external](/foundations/messages/external-out) messages are broadcast to the external network and can be read by anyone.
+
+Every internal message should have some Toncoin attached to it so that it can pay for the cost of handling it. External messages cannot have Toncoin attached to them because they spend some of their time outside the blockchain, and only the blockchain can prove the transfer of Toncoin is correct.
+
+Incoming external messages come from an external address, and outgoing external messages go to an external address.
+
+The _state_ of the account changes only when it handles messages. Messages also change the account's balance. To have a state and a balance, and to handle messages, the account must be `active`.
```

Comment:
transaction can be triggered without a message.. for example, tick-tock and due-payment transactions (this one is applicable to "regular" smart contracts that devs write)..